src/Libraries/Microsoft.Extensions.Telemetry.Abstractions/Enrichment/ILogEnricher.cs

```diff
@@ -4,7 +4,7 @@
 namespace Microsoft.Extensions.Diagnostics.Enrichment;
 
 /// <summary>
-/// Represents a component that augments log records with additional properties.
+/// Augments log records with additional properties.
```

Comment:
The xml-docs should be preserved as those were an improvement in the main, see https://github.com/dotnet/extensions/pull/4393.